### ISSUE TITLE
Instructions on Ordering and Assembling PCB?

### ISSUE THREAD

**Issue by howardt12345** [JKR-31/JK_M0nkey_Keyboard#2]
Hi, I'm Howard, and I stumbled upon your project when looking for a keyboard that has the exact same specs that the keyboard you designed has. I just want to say that you did an absolutely amazing job at designing this keyboard, it's such a nice keyboard and one I would love to have myself.

I'm looking to make this keyboard for myself. However, given my lack of experience in electrical engineering, I'm really not sure on how to use the files provided in your repository, and what I additional information I should provide to a PCB prototype manufacturer in order to get a PCB created for this keyboard.  May you please update the [readme](https://github.com/JKR-31/JK_M0nkey_Keyboard/blob/main/PCB/readme.md) in the PCB folder with instructions on how to order the PCB from a PCB prototype manufacturer such as JLCPCB or PCBWay, and how to assemble the PCB? I, and many others who might seek to build the keyboard you designed, would greatly appreciate it.

Also, it would be really nice if you were to open source the KICad files for this keyboard as well. I would love to use this project as a reference for my own potential keyboard designs if you don't mind.

**Comment by JKR-31** [comment]
Hi Howard,

cool that you found my project and liked it.
I updated the PCB directory. I added the same zip-file that I used to order the PCB from JLC-PCB and also a screenshot of all the options that i choose on my order.
Also wrote some things in the read-me about the assembly process.

As a tip if you ever design something similar with a USB-Hub. I mostly referenced my schematic on one of the demo-/evaluation-boards, that microchip sells. Most of the manufactures of these USB-Hub-chips offer these demoboards and also upload the schematics for those on there websites.
(here the link: https://www.microchip.com/en-us/development-tool/EVB-USB2513BC & http://ww1.microchip.com/downloads/en/DeviceDoc/evb2513q36bas.pdf)
